lib/link-constraints.ts

```diff
@@ -1676,6 +1676,46 @@ export const constraints: LinkConstraint[] = [
 			inverse: 'link-constraint-loop-has-product-repository',
 		},
 	},
+	{
+		slug: 'link-contraint-reaction-is-attached-to-message',
+		name: 'is attached to',
+		data: {
+			title: 'Reaction',
```

Comment:
```suggestion
			title: 'Message',
```

Just following the pattern of other link constraint definitions